### PR TITLE
DOC: stylistic consistency, typos, formatting

### PR DESCRIPTION
**Pull request by mkg33**
Stylistic revision, punctuation, typos, etc. Also, closes gh-306.

### CODE REVIEW

File: www/install.rst [hunk 1]
```diff
@@ -5,51 +5,82 @@ Installation
...
 
 Other options include:
 
-* `Enthought Canopy <https://www.enthought.com/products/canopy>`_: Free and commercial versions; Supports Linux, Windows, and Mac; Includes an IDE.
-* `Python(x,y) <https://python-xy.github.io/>`_: A free distribution including scientific packages; based around the `Spyder IDE <https://www.spyder-ide.org/>`__; Windows and Ubuntu; Py2 only.
-* `WinPython <https://winpython.github.io>`_: Another free distribution including scientific packages and the Spyder IDE; Windows only, but more actively maintained and supports the latest Python 3 versions.
-* `Pyzo <http://www.pyzo.org/>`_: A free distribution based on Anaconda and the IEP interactive development environment; Supports Linux, Windows, and Mac.
-* `NeuroDebian repository <http://neuro.debian.net>`_ : Only for the Debian OS and geared toward neuroscientists.
+* `Enthought Canopy <https://www.enthought.com/products/canopy>`_: Free and
+commercial versions; supports Linux, Windows, and Mac; includes an IDE.
```
Comment (rgommers) [inline www/install.rst]: These need an indent to line up with the first character (not the `*`) on the line above to render correctly. 

If you run `make html` you should see no warnings; this gives:
```
.../scipy.org/www/install.rst:41: WARNING: Bullet list ends without a blank line; unexpected unindent.
```

File: www/scipylib/faq.rst [hunk 2]
```diff
@@ -66,14 +66,14 @@ Python's lists are efficient general-purpose containers. They support
...
 they can contain objects of differing types mean that Python must store
-type information for every element, and must execute type dispatching
+type information for every element, and must execute type-dispatching
 code when operating on each element. This also means that very few list
 operations can be carried out by efficient C loops -- each iteration
 would require type checks and other Python API bookkeeping.
 
-What's the story with Numeric, numarray, and NumPy?
+What's the story behind Numeric, numarray, and NumPy?
 ###################################################
```
Comment (rgommers) [inline www/scipylib/faq.rst]: this needs two more `#`'s, it now causes:
```
.../scipy.org/www/scipylib/faq.rst:77: WARNING: Title underline too short
```

File: www/scipylib/faq.rst [hunk 3]
```diff
@@ -359,25 +359,25 @@ The function :func:`asmatrix` converts an array into a matrix (without ever
...
 versions of Python.
 
 How do I find the indices of an array where some condition is true?
 ###################################################################
 
-The prefered idiom for doing this is to use the function :func:`numpy.nonzero`
-, or the :meth:`nonzero` method of an array. Given an array ``a``, the
-condition ``a > 3`` returns a boolean array and since ``False`` is
+The preferred idiom for doing this is to use the function :func:`numpy.nonzero`,
+ or the :meth:`nonzero` method of an array. Given an array ``a``, the
```
Comment (rgommers) [inline www/scipylib/faq.rst]: space at start of this line needs removing

File: www/topical-software.rst [hunk 4]
```diff
@@ -58,126 +58,126 @@ Tutorials and texts
...
 - `Plotly <https://plot.ly/python/>`__: an online Python environment for data exploration and graphing. Plotly has a command line, and allows for storage and sharing of Python scripts, and has special support for `interactive Plotly graphs <https://plot.ly/python/>`__.
-- `Wakari <https://wakari.io/>`__: Scientific and Technical Computing in the browser.  You can create and share workflows, IPython notebooks, plots, and applications in the cloud all on-top of Anaconda a complete Scientific and Big-Data Python distribution.   More information available at the `Wakari website <https://wakari.io/>`__.
-- `Other IDE links <https://wiki.python.org/moin/IntegratedDevelopmentEnvironments>`__: the official Python website maintains a comprehensive lists of Integrated Development Environments for Python.
+- `Wakari <https://wakari.io/>`__: scientific and technical computing in the browser. You can create and share workflows, IPython notebooks, plots, and applications in the cloud all on-top of Anaconda a complete scientific and Big Data Python distribution. More information available on the `Wakari website <https://wakari.io/>`__.
```
Comment (rgommers) [inline www/topical-software.rst]: Could you delete this? Wakari is no longer a thing

File: www/topical-software.rst [hunk 5]
```diff
@@ -58,126 +58,126 @@ Tutorials and texts
...
 - `Pyrex <http://www.cosc.canterbury.ac.nz/greg.ewing/python/Pyrex/>`__: Pyrex lets you write code that mixes Python and C data types any way you want, and compiles it into a C extension for Python. See also `Cython <http://cython.org>`__.
 - `PyCxx <http://cxx.sourceforge.net>`__: CXX/Objects is a set of C++ facilities to make it easier to write Python extensions. The chief way in which PyCXX makes it easier to write Python extensions is that it greatly increases the probability that your program will not make a reference-counting error and will not have to continually check error returns from the Python C API.
 - `ctypes <http://starship.python.net/crew/theller/ctypes>`__: a package to create and manipulate C data types in Python, and to call functions in dynamic link libraries/shared dlls. It allows wrapping these libraries in pure Python.
 - `railgun <http://tkf.bitbucket.org/railgun-doc/>`__: ctypes utilities for faster and easier simulation programming in C and Python
 
-Wrapping Matlab, R and IDL Codes
+Wrapping MATLAB, R, and IDL codes
 --------------------------------
```
Comment (rgommers) [inline www/topical-software.rst]: need one more `-` here, underline too short warning now

Comment (mkg33) [reply www/topical-software.rst]: Thanks for the careful review and apologies for the silly mistakes :-)

**Comment by rgommers** [comment]
Merged, thanks @mkg33

**Pull request merged**